modules/microshift-install-bootc-workflow.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// microshift_install_bootc/microshift-about-rhel-image-mode
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-install-rhel-image-mode-conc_{context}"]
+= The image mode for {op-system-base} with {microshift-short} workflow
+
+To use image mode for {op-system-base}, ensure that the following resources are available:
+
+* A {op-system-base} {op-system-version} host with an active Red Hat subscription for building {microshift-short} bootc images.
+* A remote registry for storing and accessing bootc images.
+* You can use image mode for RHEL with a {microshift-short} cluster on aarch64 or x86_64 system architectures.
+
+The workflow for using image mode with {microshift-short} includes the following steps:
+
+. Find and use a prebuilt {microshift-short} container image to install {op-system-base-full}.
+. Build a custom {microshift-short} container image if the prebuilt {microshift-short} container image requires customization.
+. Run the container image.
+
+[IMPORTANT]
+====
+The `rpm-ostree` file system is not supported in image mode and must not be used to make changes to deployments that use image mode.
```

Comment:
```suggestion
The `rpm-ostree` file system is not supported in image mode. Do not use the `rpm-ostree` file system to modify deployments that use image mode.
```

This is up to you. Apologies if I nitpicked.

Reply:
That's a good call. It will translate better.